vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -233,7 +233,12 @@ type internal FSharpLanguageService(package : FSharpPackage) as this =
         let site = siteProvider.GetProjectSite()
         let projectGuid = Guid(site.ProjectGuid)
         let projectFileName = site.ProjectFileName()
-        let projectId = workspace.ProjectTracker.GetOrCreateProjectIdForPath(projectFileName, projectFileName)
+
+        let projectDisplayName = 
```

Comment:
Mind adding a comment explaining why `projectFileName` doesn't suffice as the `projectDisplayName`?  Looking at this as a newcomer, I'd be surprised at why this code is needed.